readme.md

```diff
@@ -1,13 +1,69 @@
+
+We must make our data
+
+- **self-descriptive** (referencing human- & machine-readable documents on how to parse & interpret it),
+- **globally precise** (using IDs that accurately identify transportation infrastructure *without local or operator-specific context*),
+- **redistributable** (being in the [public domain](https://en.wikipedia.org/wiki/Public_domain), allowing for easy chunking, replication & storage).
+
+We must make our APIs
+
+- **openly documented** (using open standards),
+- **publicly available** (without authentication, in the internet, in a best-effort approach),
+- **federated** (linking to other APIs instead of aggregating them).
+
+We must develop our **data standards** in the open (allowing barrier-free participation & collaboration) and **freely licensed** (to enable wide-spread use). They **should cleanly separate semantics, replication/transport, storage & encoding**. They should not reinvent the wheel ([NeTEx](https://en.wikipedia.org/wiki/NeTEx)), but **rely on existing de-facto standards** (such as [GTFS](https://gtfs.org)) where applicable.
+
+### Stable Identifiers
+
+By nature, public Transportation data has many links, because it reflects strongly interconnected public transportation systems. **When data by an author/source "A" refers to data from *another* author/source "B", it needs a reliable and precise way to identify items in "B" data.** In federated systems, especially in [linked data](https://en.wikipedia.org/wiki/Linked_data) systems, the need for stable & globally unique IDs becomes even more important than in the traditional centralized systems.
+
+*Note:* The aforementioned [*Publishing Transport Data for Maximum Reuse*](https://phd.pietercolpaert.be) has a specific [section on stable identifiers for interoperable data](https://phd.pietercolpaert.be/chapters/measuring-iop).
+
+**This project explores how to derive such IDs *from the data itself* in a deterministic way**. **There is an inherent trade-off**: In order to prevent collisions, the input data (which the ID will be computed from) must be quite detailed; On the other hand, for these IDs to be easily computable (e.g. offline), only little data should have to be transferred & stored.
+
+It is an ongoing process of
+
+- generalising them enough to support all relevant kinds of public transportation infrastructure,
+- desgining them to identify local infrastructure precisely (enough),
+- finding and "covering" enough edge cases for the standard to be practical in real-world szenarios,
+- formally describing, testing and implementing the standard in multiple evironments & programming languages.
+
+**In addition, we use indeteministic but well-known (and thus stable) identifiers**, such as [Wikidata](https://wikidata.org/) or [OSM](https://openstreetmap.org/) IDs, to work as a "stepping-stone" until the deterministic IDs have widespread adoption.
```

Comment:
Wikidata IDs are however more stable. OpenStreetMap identifiers can change when updates happen due to the geospatial nature of the data. That’s why it’s OSM linking to wikidata, and hardly the other way around.

Reply:
Fair enough, will remove OSM.

(Keep in mind that you would mostly link to a more higher-level OSM `relation` which changes less often than the `way`s and `node`s it consists of.)

Reply:
Don’t think it should be removed necessarily. I like the idea of linking it to higher level ways/nodes/relations!